### PR TITLE
💡 Add docstrings for podcasts

### PR DESCRIPTION
**Pull request by tcmetzger**
This pull request adds docstrings to several files related to podcast scrapers.

### CODE REVIEW

**Comment by tcmetzger** [comment]
I'D appreciate some help with okr/scrapers/podcasts/spotify_api.py - I could use some info on the types returned by the various methods there .

File: okr/scrapers/podcasts/__init__.py [hunk 1]
```diff
@@ -1,592 +1,670 @@
...
+        podcasts = podcasts.filter(podcast_filter)
+
+    for podcast in podcasts:
+        print("Scraping spotify API for", podcast)
+
+        if not podcast.spotify_id:
+            print("No Spotify ID for", podcast)
+            continue
+
+        # Retrieve follower and listener data for podcast (ignores start_date)
```
Comment (jh0ker) [inline okr/scrapers/podcasts/__init__.py:199]: Only follower data ignores `start_date`. Reason is that the API only supports querying the current follower number, no historic data is provided.

**Review by jh0ker — CHANGES_REQUESTED** [review]
I left some comments.
 
Also, what happened in commmit c73063e3fcd92fef2e00828b6ce6270ea0345390 ? GitHub shows me the whole file as changed. Is that because of something like different line endings or is there really that much changed that it sees it as a completely new file?

File: okr/scrapers/podcasts/feed.py [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+def parse(url: str) -> feedparser.util.FeedParserDict:
+    """Parse feed from url into FeedParserDict.
+
+    Args:
+        url (str): Url to be parsed.
+
+    Returns:
+        feedparser.util.FeedParserDict: Parsed data.
+    """
+    return feedparser.parse(url)
```
[the remark this answers is not in the document]
Comment (tcmetzger) [reply okr/scrapers/podcasts/feed.py:16]: Voll!

**Review by jh0ker — APPROVED** [review]

**Pull request merged**